nvflare/app_common/workflows/model_controller.py

```diff
@@ -63,7 +63,7 @@ def __init__(
             raise ValueError("task_check_period must be greater than 0.")
         self._task_check_period = task_check_period
         self._persistor_id = persistor_id
-        self._persistor = None
+        self.persistor = None
```

Comment:
I made this change to standardize since all of our existing controllers used `self.persistor` rather than `self._persistor`. This will help in the optional case of distributing the final results, as we can check for a controller’s `self.persistor` attribute.